### PR TITLE
Fix scene switching + Fixed random stuff too!

### PR DESCRIPTION
**Pull request by po-trottier**
Currently when you complete the level (get the relic and go through the door) we will load the demo scene again and you can start fresh again! You can do it as many times as you want without errors.

Merge #90 before looking at this PR! It will reduce the # of files changed a lot :)

### CODE REVIEW

**Review by daniela-venuta — APPROVED** [review]

File: Hush/ProjectSettings/TagManager.asset [hunk 1]
```diff
@@ -22,7 +23,7 @@ TagManager:
   - UI
   - Ground
   - Obstacle
-  - Wall
```
Comment (emilie-martin) [inline Hush/ProjectSettings/TagManager.asset:25]: any reason we're removing the Wall tag?

**Review by emilie-martin — APPROVED** [review]
just a couple of questions but looks good!

File: Hush/ProjectSettings/TagManager.asset (hunk 1 above)
Comment (po-trottier) [reply Hush/ProjectSettings/TagManager.asset:25]: Use Obstacle instead

File: Hush/Assets/Scripts/Weapon/Spell.cs [hunk 2]
```diff
@@ -1,48 +1,66 @@
...
-    public void PerformSpecialAttack()
+    public void PerformHeavyAttack(InputAction.CallbackContext context)
     {
-        CurrentDamage = SPECIAL_DAMAGE;
         animator.SetTrigger(PlayerAnimator.SpellSpecialAttack);
-        CreateSpellAttack(CurrentDamage);
+        CreateSpellAttack(AttemptCrit(HeavyDamage));
+    }
+
+    public float AttemptCrit(float damage)
```
[the remark this answers is not in the document]
Comment (po-trottier) [reply Hush/Assets/Scripts/Weapon/Spell.cs:52]: Yeah should add a precision on that line in a future PR

File: Hush/Assets/Prefabs/Environment/Domes/Dome.prefab [hunk 3]
```diff
@@ -11,13 +11,13 @@ GameObject:
...
   - component: {fileID: 2311186444272300722}
-  - component: {fileID: 7786904841737782451}
+  - component: {fileID: 4107575512628067778}
   - component: {fileID: 6917358588575371276}
   - component: {fileID: 4816975685772881064}
   - component: {fileID: 3994485805967114560}
-  m_Layer: 9
+  m_Layer: 7
   m_Name: Dome
-  m_TagString: Relic Dome
```
[the remark this answers is not in the document]
Comment (po-trottier) [reply Hush/Assets/Prefabs/Environment/Domes/Dome.prefab:20]: Good call, TODO on a future PR

**Pull request merged**